bundle/libraries/local_path.go

```diff
@@ -95,7 +95,7 @@ func containsPipFlag(input string) bool {
 	// Trailing space means the the flag takes an argument or there's multiple arguments in input
 	// Alternatively it could be a flag with no argument and no space after it
 	// For example: -r myfile.txt or --index-url http://myindexurl.com or -i
-	re := regexp.MustCompile(`(^|\s+)--?[a-zA-Z0-9-]+(\s+|$)`)
+	re := regexp.MustCompile(`(^|\s+)--?[a-zA-Z0-9-]+(([\s|=]+)|$)`)
```

Comment:
Nit: I think you only need `([\s=]|$)` -- either a space, an `=`, or the end of the string.

More than one space or `=` doesn't change the result.